cpp/ps0003/problem 3 s.cpp

```diff
@@ -1,14 +1,18 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
 int main()
 {
-int number;    
-cout <<"Enter any number\n";
-cin >> number;
-if(number % 2 ==0)
-cout << " number is even \n";
-else
-cout << " number is odd \n" << endl;
+  // prompting user for number 2/2
+  int number;    
+  cout << "Enter any number\n";
+  cin >> number;
 
-return 0;
-}
\ No newline at end of file
+  // checking if the number is odd or even 3/3
+  if(number % 2 ==0)
+      cout << " number is even \n";
+  else
+      cout << " number is odd \n" << endl;
+  
+  return 0;
+}
+// great work 😎 9/10 ⭐
```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code

you can use ternary operator
```cpp
std::cout << "Number is " << ((x % 2 == 0) ? "even" : "odd") << std::endl
```